src/imitation/fenchel_young.jl

```diff
@@ -0,0 +1,111 @@
+"""
```

Comment:
For consistency, I would rename this file `fenchel_young_loss.jl`.
Also, I would name the folder `imitation_losses`, or maybe it's confusing with the `ImitationLoss` struct ?